src/Apps/W1/Shopify/App/src/Order Fulfillments/Codeunits/ShpfyFulfillmentOrdersAPI.Codeunit.al

```diff
@@ -207,4 +278,13 @@ codeunit 30238 "Shpfy Fulfillment Orders API"
         else
             exit(Enum::"Shpfy Delivery Method Type"::" ");
     end;
+
+    local procedure ConvertToRequestStatus(Value: Text): Enum "Shpfy FF Request Status"
+    begin
+        Value := CommunicationMgt.ConvertToCleanOptionValue(Value);
+        if Enum::"Shpfy FF Request Status".Names().Contains(Value) then
+            exit(Enum::"Shpfy FF Request Status".FromInteger(Enum::"Shpfy FF Request Status".Ordinals().Get(Enum::"Shpfy FF Request Status".Names().IndexOf(Value))))
```

Comment:
Not for this PR, but we need devtools to give us a better way to deal with enums. This is just an abomination.